CONTRIBUTING.rst

```diff
@@ -41,7 +41,11 @@ DateParser could always use more documentation, whether as part of the
 official DateParser docs, in docstrings, or even on the web in blog posts,
 articles, and such.
 
-After you make local changes to the documentation, build it with ``tox``::
+After you make local changes to the documentation, you should install ``tox``::
+
+    pip install tox
+
+Then, you will be able to build the project running::
```

Comment:
While I don’t expect people to do it, this reads like you need to install `tox` every time.

What about adding a `.. note::` below that explains that, if you do not have `tox` installed, you need to install it first using `pip install tox`?